### ISSUE TITLE
Support query to determine largest metrics

### ISSUE THREAD

**Issue by willchan** [GoogleCloudPlatform/prometheus-engine#196]
I was trying to figure out what my biggest metrics were, per https://www.robustperception.io/which-are-my-biggest-metrics.

So I tried `topk(10, count by (__name__)({__name__=~".+"}))`, which failed with `Error executing query: invalid parameter "query": substitute queries failed: convert vector selector failed: series selector must specify an exact metric name`.

I'm not sure what part failed exactly, but I'm guessing it's the `__name__=~".+"` label selector. I'm guessing that might be hard to do in Monarch, so I don't know if it's reasonable to support that directly, or if there's a better way to support a way to determine which metrics are most expensive.

**Comment by lyanco** [comment]
Yeah - we currently don't support wildcard metric name matching. This is on the roadmap as part of our path to 100% query conformance, but it's not likely to be done for some time.

**Comment by lyanco** [comment]
That being said, you can look at the Cloud Monitoring Metrics Diagnostics page for this.

https://cloud.google.com/monitoring/custom-metrics/diagnostics

It has that exact info! You can also see the GMP-specific instructions here: https://cloud.google.com/stackdriver/docs/managed-prometheus/cost-controls
